pkg/sfu/streamtracker/streamtracker_dd.go

```diff
@@ -126,15 +129,28 @@ func (s *StreamTrackerDependencyDescriptor) SetPaused(paused bool) {
 		return
 	}
 	s.paused = paused
+
+	var notifyFns []func(status StreamStatus)
+	var notifyStatus StreamStatus
 	if !paused {
 		s.resetLocked()
+
+		notifyStatus = StreamStatusStopped
+		for _, f := range s.onStatusChanged {
+			notifyFns = append(notifyFns, f)
+		}
```

Comment:
`notifyFns = append(notifyFns, s.statusChanged[:]...)`?

Reply:
Yes, of course, changing :-)